howto/versioning-documentation.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Technical Documentation Versioning with Hugo & Netlify
+author: [Nate Waddington](https://github.com/nate-double-u)
+date: 2021-01-30
+update: 2021-07-05
+license: CC-BY-4.0
+---
+
```

Comment:
This is an artifact of the slideshow markdown file. We can remove it if we don't want it.

Reply:
Please remove it.